jmx-metrics/src/test/java/io/opentelemetry/contrib/jmxmetrics/OtelHelperSynchronousMetricTest.java

```diff
@@ -0,0 +1,466 @@
+/*
+ * Copyright The OpenTelemetry Authors
```

Comment:
Figured it's ok not to copy in the memoization tests here since we don't do anything special to memoize sync measurements